homeassistant/components/nextcloud/config_flow.py

```diff
@@ -21,6 +28,13 @@
         vol.Required(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): bool,
     }
 )
+DATA_SCHEMA_REAUTH = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
```

Comment:
Should username really be prompted again ?
Should it not be instead added to the placeholders?

Makes me think also that username maybe should be part of `_async_abort_entries_match`?

Reply:
i don't think, user should be part of `_async_abort_entries_match`, because the created sensors are system (_url_) specific (_not user specific_)
Further it may happen, that the user wants use a new nextcloud-user for monitoring purposes, so it should be possible to change also the username, to be able to keep the sensors history data.

Reply:
As long as it doesn't request any user specific data it should be fine then.